crates/sui/tests/snapshots/ptb_files_tests__assign_valid.ptb.snap

```diff
@@ -21,8 +21,6 @@ expression: "results.join(\"\\n\")"
 │ assign        │ c0 c.0           │
 │ assign        │ c1 c.1           │
 ├───────────────┼──────────────────┤
```

Comment:
Does this snapshot imply that if a gas budget isn't supplied, the `--preview` output will not "end" properly? (it will have these unterminated box-drawing characters?

Reply:
Fixed!